### PR TITLE
Add helper-guidance

### PR DESCRIPTION
**Pull request by rattus-rattus-rattus**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ defaults:
 
 locale: en_US
 
-plugins: []
+plugins: []
```
Comment (mniip) [inline _config.yml]: pls fix newline

File: helper-guidance.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+There are times where, despite your best efforts, a troublesome user will continue to cause issues. In cases like this, it is often best for everyone involved that you take a step back and remove yourself from the situation. Always remember - you are volunteering to help somebody with their mathematics problems - not to put up with verbal abuse. If you feel uncomfortable in a situation, leave. Remember to use Discord's block tool to block users if you think it is appropriate.
+
+### Unacceptable behavior
+Unfortunately, there are occasions where users come to the server with the sole intent of causing problems, stirring up trouble, or otherwise trolling. While most bad behavior can be safely ignored, there are several things that are unacceptable to do or say on the server. For a full breakdown of unacceptable behaviors, please refer to [#rules](https://discord.com/channels/268882317391429632/486128019652476939){: .mention target="_blank"} and/or the [Discord Terms of Service](https://discord.com/terms){: .mention target="_blank"}. If you see any of this type of behavior, you should ping <a class="mention">@Moderators</a> immediately and remove yourself from the situation.
```
Comment (mniip) [inline helper-guidance.md]: #rules could link to `/rules` instead (probably not `target="blank" in that case).

Discord Terms of Service should not be `.mention`.

**Review by mniip — CHANGES_REQUESTED** [review]

**Review by mniip — APPROVED** [review]
LGTM on the technical side

File: helper-guidance.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Dealing with difficulties
+
+### Language barriers
+While the main language of the server is English, we are a diverse community with users from all backgrounds. While it is nice when a question is asked in perfect English, be mindful that there are many reasons why a user may make mistakes or be unclear in their writing. They may speak English as a second language, be typing on a phone keyboard, or have a condition that makes it difficult for them to type.
+
+If someone's writing is unclear, it is best to calmly and politely ask them if they can rephrase their question.
+
+### Bad language
+While it is not against server rules to use bad language, it may sometimes be frustrating to see. Remember that we are here to discuss and learn mathematics, not to correct eachother's words and use of language. Unless their language use is making you very uncomfortable or is [breaking server rules](#unacceptable-behavior), try to ignore bad words and language and think about maybe [stepping back](#when-to-step-back).
```
Comment (riemann-discord) [inline helper-guidance.md]: each other's

File: helper-guidance.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+### Rudeness
+Not everyone has perfect manners. Some users are, unfortunately, rude to other users. It is rarely the case that calling users out on this improves the situation; worse still, being rude to a user who is being rude to you invariably leads to total breakdown in communication.
+
+Nevertheless, it is extremely important not to automatically assume rudeness. Users may be from a different cultural background to you and/or have [language barriers](#language-barriers) that can sometimes make it very difficult to judge tone and intent. If you aren't sure, give users the benefit of the doubt.
+
+## When things go wrong
+
+### When to step back
+There are times where, despite your best efforts, a troublesome user will continue to cause issues. In cases like this, it is often best for everyone involved that you take a step back and remove yourself from the situation. Always remember - you are volunteering to help somebody with their mathematics problems - not to put up with verbal abuse. If you feel uncomfortable in a situation, leave. Remember to use Discord's block tool to block users if you think it is appropriate.
```
Comment (riemann-discord) [inline helper-guidance.md]: In cases like these*

**Review by riemann-discord — APPROVED** [review]

**Pull request merged**